MathPortal/MathPortal/Source/Custom/Custom Views/KeyboardView/KeyboardView.swift

```diff
@@ -167,19 +205,41 @@ class KeyboardView: UIView {
     }
 }
 
-// MARK: - custom layout setup
+// MARK: - Custom layout setup
 
 extension KeyboardView {
     
     private func setupLettersKeyboard() {
+        
+        [greekLettersButton, upperLettersButton].forEach { button in
```

Comment:
It's nice to lose optionals in such cases. There are 2 nice ways to do so.

Either use inside block `guard let button = button else { return }`
Or compact map it `[greekLettersButton, upperLettersButton].compactMap { $0 }.forEach`